tests/unit/test_create_nodes.py

```
@@ -1,32 +1,25 @@
 import asyncio
-from typing import Callable, List, Optional
+from typing import Callable, List, Optional, Awaitable
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import

(F401)